pkg/util/pki/common.go

```diff
@@ -57,6 +58,8 @@ const (
 	// KafkaUserAnnotationName used in case of PKIbackend is k8s-csr to find the appropriate kafkauser in case of
 	// signing request event
 	KafkaUserAnnotationName = "banzaicloud.io/owner"
+	// MaxCertManagerCNLen specifies the maximum common name supported by cert-manager
+	MaxCertManagerCNLen = 64
```

Comment:
The max length limit on CN field I think is rather imposed by certificate standards and not cert-manager this comment should be updated accordingly (see https://docs.digicert.com/manage-certificates/public-certificates-data-entries-that/#64character-maximum-limit-violation)

Reply:
will update the var and comment to reflect this